### DISCUSSION TITLE
How to fix this form

### DISCUSSION THREAD

**Discussion by Unknowncoder231** [emailjs-com/help#164] [Q&A]
I have created this form and whenever I click submit I got error faild to submit please try again 
store url:https://store1.ecomsmartify.shop/pages/book-an-appointment
store password: 4

<div class="booking-page">
    <div class="form-container">
        <form id="booking-form" method="POST">
            <label for="first_name">First Name</label>
            <input type="text" id="first_name" name="first_name" placeholder="First Name" required>

            <label for="last_name">Last Name</label>
            <input type="text" id="last_name" name="last_name" placeholder="Last Name" required>

            <label for="contact_number">Contact Number</label>
            <input type="tel" id="contact_number" name="contact_number" placeholder="Contact Number" required>

            <label for="email">Email</label>
            <input type="email" id="email" name="email" placeholder="Email" required>

            <label for="address">Address</label>
            <input type="text" id="address" name="address" placeholder="Address">

            <label for="sets">Number of Sets</label>
            <input type="number" id="sets" name="sets" min="0" value="0">

            <label for="message">Message</label>
            <textarea id="message" name="message" placeholder="Message"></textarea>

            <button type="submit">Submit</button>
        </form>
    </div>
    <div class="image-container">
        <img src="https://cdn.shopify.com/s/files/1/0882/9612/8818/files/54.jpg?v=1733382319" alt="Booking Image">
    </div>
</div>

<script src="https://cdn.emailjs.com/dist/email.min.js"></script>
<script>
    (function() {
        // Initialize EmailJS with your User ID
        emailjs.init("-LoCHKqN0kWHH_E6w");
    })();

    // Attach event listener to the form
    const form = document.getElementById('booking-form');
    form.addEventListener('submit', (e) => {
        e.preventDefault(); // Prevent the default form submission

        // Collect form data
        const formData = new FormData(form);
        const data = {};
        formData.forEach((value, key) => {
            data[key] = value;
        });

        // Send the email using EmailJS
        emailjs
            .send("service_bhji3ve", "template_52ou8hc", data)
            .then(() => {
                alert("Form submitted successfully!");
                form.reset(); // Clear the form after successful submission
            })
            .catch((error) => {
                console.error("EmailJS Error:", error);
                alert("Failed to submit the form. Please try again.");
            });
    });
</script>

**Comment by xr0master** [comment]
1. check the error message
2. fix the error

I suggest you use an official documentation
